pkgs/top-level/all-packages.nix

```diff
@@ -14053,9 +14053,26 @@ in
 
   libsForQt512 = recurseIntoAttrs (lib.makeScope qt512.newScope mkLibsForQt5);
 
+  qt514 = recurseIntoAttrs (makeOverridable
+    (import ../development/libraries/qt-5/5.14) {
+      inherit newScope;
+      inherit stdenv fetchurl fetchpatch fetchFromGitHub makeSetupHook makeWrapper;
+      inherit bison;
+      inherit cups;
+      inherit dconf;
+      inherit harfbuzz;
+      inherit libGL;
+      inherit perl;
+      inherit gtk3;
+      inherit (gst_all_1) gstreamer gst-plugins-base;
+      inherit llvmPackages_5;
+    });
+
+  libsForQt514 = recurseIntoAttrs (lib.makeScope qt514.newScope mkLibsForQt5);
+
   # TODO bump to 5.12 on darwin once it's not broken
```

Comment:
Has this been addressed?

Reply:
I don't know where the qt version for darwin is set, and I don't have a darwin machine to test this on.

Reply:
Since we don't have any older version of Qt packaged in nixpkgs anymore I assume this was already done in a previous change. I will remove the comment, but it would still be great if someone could test the new version on darwin.